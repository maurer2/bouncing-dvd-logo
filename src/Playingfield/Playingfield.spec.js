import React from 'react';
import 'jest-styled-components';

import PlayingField from './Playingfield';

describe('PlayingField', () => {
  it.todo('should match snapshot');
  // const subscribeMock = jest.fn();
  // const unsubscribeMock = jest.fn();

  // const context = {
  //   loop: {
  //     subscribe: subscribeMock,
  //     unsubscribe: unsubscribeMock,
  //   },
  // };

  // const wrapper = shallow(<PlayingField isPaused={false} />, { context });

  // wrapper.update();

  // test('should match snapshot', () => {
  //   expect(toJson(wrapper)).toMatchSnapshot();
  // });

  // test('should call loop.subscribe', () => {
  //   expect(subscribeMock).toBeCalled();
  // });

  // test('should call loop.unsubscribe', () => {
  //   wrapper.unmount();
  //   expect(unsubscribeMock).toBeCalled();
  // });
});
