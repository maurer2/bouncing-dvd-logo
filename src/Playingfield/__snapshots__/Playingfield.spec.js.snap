// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Playfield should match snapshot 1`] = `
<PlayfieldWrapper>
  <Logo
    changeColours={false}
    colours={
      Array [
        "white",
        "red",
        "blue",
        "yellow",
        "fuchsia",
        "lime",
      ]
    }
    height={138}
    width={150}
  />
  <Sound
    playSound={false}
  />
</PlayfieldWrapper>
`;
